runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/CombineTranslationTest.java

```diff
@@ -179,6 +185,12 @@ public void leaveCompositeTransform(Node node) {
     }
   }
 
+  private static Coder<?> getAccumulatorCoder(
+      CombinePayload payload, RehydratedComponents components) throws IOException {
```

Comment:
Not something to necessarily change here, but do we still need `RehydratedComponents` at this point? It depends how it's used in general, but I suspect that using the components may be sufficient (in most cases).

Reply:
RehyrdatedComponents is still useful as an object cache for coders/windowing strategy/...

Still undecided what to do in this space.

Reply:
Ack